rental.rb

```diff
@@ -0,0 +1,13 @@
+class Rental
+  attr_accessor :date
+
+  def initialize(date, person, book)
+    @date = date
+
+    @person = person
+    person.rentals << self
+
+    @book = book
+    book.rentals << self
+  end
+end
```

Comment:
- [ ] In the rental class , The Getters and Setters for @book and @person instance variables should be present.